package.json

```diff
@@ -19,6 +19,7 @@
   ],
   "atomShellVersion": "0.21.0",
   "dependencies": {
+    "6to5-core": "3.0.14",
```

Comment:
Might be a good idea to have this as `^3.0.14` as I push out releases pretty often and don't want atom users to be disadvantaged.